tests/tests.c

```diff
@@ -207,14 +207,274 @@ TT_TEST(ParseInvalidStrings)
 
 /* ------------------------------------------------------------------------- */
 
+TT_TEST(BigInt1)
+{
+	char *buf;
+	FILE *f;
+	config_t cfg;
+	int rc;
+	int ival;
+	long long llval;
+
+	/* int = 5 */
+	buf = "someint=5;";
+
+	f = fmemopen(buf, strlen(buf), "r");
```

Comment:
I am not sure how portable fmemopen() is. Why not just use config_read_string() below?